sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -177,6 +181,14 @@ export class ManagedIdentityCredential implements TokenCredential {
         );
       }
     }
+
+    logger.info(`Using ${managedIdentitySource} managed identity.`);
+
+    // Check if either clientId, resourceId or objectId was provided and log the value used
```

Comment:
We should check with other languages to see if they're also logging resource ID and object ID. I was under the impression that we were only logging client ID.